wiki/Ranking_Criteria/en.md

```diff
@@ -86,7 +87,7 @@ Note: If a mapset's song contains matching sections to the TV size song, in the
 ### Rules
 - **No two hit objects can be placed on the same tick** This includes hit circles, starts and ends of sliders, and starts and ends of spinners. osu!mania maps are exempt from this.
 - **There must not be any obscene imagery in the background/storyboard/video content.** This includes nudity, near-nudity, sexual references, violence, drug abuse, etc. Keep things PG, suitable for ages 12+. Images should be on a level that can be displayed on all-audience TV, on public signage, and of nature that does not require censoring in any country.
```

Comment:
`Keep things PG, suitable for ages 12+.` -> `Keep things PG (suitable for ages 12+).`?